src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the import path for the `Person` component is correct. The path './components/Person/Person' suggests that there is a directory named `Person` containing an `index.js` or `index.jsx` file, or a file named `Person.js` or `Person.jsx`. Verify that this path is correct and that the `Person` component is properly exported from the specified location.